### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by Sangariga** [SebastianMC/obsidian-custom-sort#90] [Q&A]

**Comment by SebastianMC** [comment]
Hi @Sangariga 

should I remove the discussion?
Is your original question no longer valid / applicable?

**Comment by Sangariga** [comment]
Hi!

Kind of waited for an answer - but then meanwhile I found two other 
soulutions that are more easy for ordering the list.
You can erase the question - just couldn`t see how I would do that.
You have a nice plugin though - for the noprogrammer crowd like myself 
it must get more simpler.
As in just write down the list in a form or change the order manually - 
in the settings maybe.
But for the others I guess this is no problem.

Have a nice day!;)

Regards!


Sangariga

---------------
